opentelemetry-api/src/logs/record.rs

```diff
@@ -79,6 +80,38 @@ pub enum AnyValue {
     Map(OrderMap<Key, AnyValue>),
 }
 
+impl fmt::Display for AnyValue {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
nit - change the variable name to say `f` instead of `fmt` for better readability. A `fmt` is also a module.